.github/workflows/edu-s3-trigger-cronjob.yml

```diff
@@ -0,0 +1,76 @@
+---
+name: EDU Trigger Cronjob
+
+on:
+  workflow_call:
+    inputs:
+      CLUSTER:
+        required: true
+        type: string
+      NAMESPACE:
+        required: true
+        type: string
+      BUCKET:
+        required: true
+        type: string
+    secrets:
+      token:
+        required: true
+      KUBECONFIG:
+        required: true
+
+jobs:
+  run_ansible_playbook_edusharing:
+    environment: ${{ inputs.CLUSTER }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          path: "repo/infra-deploy"
+          repository: hpi-schul-cloud/infra-deploy
+
+      - name: Create Kubeconfig file
+        working-directory: ${{ github.workspace }}/
+        run: |
+          echo "${{ secrets.KUBECONFIG }}" > kubeconfig
+
+      - name: Install Kubectl
+        working-directory: ${{ github.workspace }}/
+        run: |
+            curl -LO https://dl.k8s.io/release/v1.23.16/bin/linux/amd64/kubectl
```

Comment:
What do we do if the kubernetes version is updated?

Reply:
will be solved in another ticket